<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ventana</class>
 <widget class="QMainWindow" name="Ventana">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proyecto Complejidad</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Ventana{
    background-color:#424242;
}

#miframe{
    background-color: #757575;
    border-radius: 7px;
}

QLabel, QPushButton, QTextEdit{
    font-family: Roboto;
    color: #fff;
}

#label_titulo{
    font-size: 24px;
    color: #81c784;
}

#label_archivo{
    font-size: 12px;
}
#label_resultados{
    font-size: 15px;
    border: 2px solid #9e9e9e;
    border-radius: 4px;
}

QPushButton{
    background-color: #81c784;
    border-radius: 4px;
    font-size: 15px;
}

QPushButton:pressed{
    background-color: #66bb6a;
}

QTextEdit{
    border-radius: 4px;
    background-color: #9e9e9e;
	font-size: 13px;
}

QTextEdit:hover{
    background-color: #bdbdbd;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="miframe">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>541</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_titulo">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Aviones y Aldeas ...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_cargar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Cargar Archivo</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>D:/ProjectosPersonales/ProyectoComplejidad/icons/carpeta.png</normaloff>D:/ProjectosPersonales/ProyectoComplejidad/icons/carpeta.png</iconset>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_resultados">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>301</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_archivo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_resultados">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Resultados</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_simplex">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>290</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Simplex</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_branch">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>290</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Branch</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_borrar">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>290</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Borrar</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_borrar</sender>
   <signal>clicked()</signal>
   <receiver>text_resultados</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
